### ISSUE TITLE
Broken images and color issues

### ISSUE THREAD

**Issue by elvin-aslanov** [oliversalzburg/superuser-dark-theme#2]
I installed it by copying and pasting [it](https://raw.githubusercontent.com/oliversalzburg/superuser-dark-theme/master/chrome-stylebot.css) using Stylish extension on Google Chrome.
![sesu](https://cloud.githubusercontent.com/assets/10452206/8442805/acdaf474-1f9a-11e5-9589-df823556c501.png)
using

**Comment by oliversalzburg** [comment]
Yeah, okay, the recent changes definitely broke this theme. I'll work on fixing it tonight.

**Comment by oliversalzburg** [comment]
@elvin-aslanov Check out https://github.com/oliversalzburg/superuser-dark-theme/blob/master/deprecated/chrome-stylebot.css This should be a lot better.

I'll try to take a bit of care of the source now...
